applications/nrf5340_audio/src/bluetooth/bt_stream/bt_le_audio_tx/bt_le_audio_tx.c

```diff
@@ -257,13 +268,14 @@ int bt_le_audio_tx_send(struct le_audio_tx_info *tx, uint8_t num_tx,
 		}
 		common_interval = tx[i].cap_stream->bap_stream.qos->interval;
 
+		struct net_buf *audio_buf = audio_frame->data;
 		/* Check if same audio is sent to all channels */
-		if (enc_audio.num_ch == 1) {
-			ret = iso_stream_send(enc_audio.data, data_size_pr_stream, tx[i].cap_stream,
-					      tx_info, common_tx_sync_ts_us);
+		if (num_ch == 1) {
+			ret = iso_stream_send(audio_buf->data, data_size_pr_stream,
+					      tx[i].cap_stream, tx_info, common_tx_sync_ts_us);
 		} else {
 			ret = iso_stream_send(
-				&enc_audio.data[(data_size_pr_stream * tx[i].audio_channel)],
+				&audio_buf->data[(data_size_pr_stream * tx[i].audio_channel)],
```

Comment:
You should be able to do something like 

```suggestion
				net_buf_pull_mem(audio_buf, data_size_pr_stream),
```

assuming that the data in `audio_buf` is ordered